.better-commits.json

```diff
@@ -1,19 +1,80 @@
 {
```

Comment:
Thanks @AlexHodson I appreciate the help. :)

Everything looks good. But you don't need to change this file. This is the config for *this* repository.

Anything that is not supplied gets replaced by the defaults at run-time. So you should see your change regardless of adding it here.

Reply:
Ahh, I see. I must have run the "init" npm script which caused the changes. I have reverted the better-commits.json file back to its original state.

Reply:
Yup no worries, great job and thanks again! Merged.